src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: 1525046400000,
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    onSelect: () => {},
+  };
+  it("renders correctly", () => {
```

Comment:
Its common convention to leave a clear line between each test to help with readability, should add one in here